crates/solvers/openapi.yml

```diff
@@ -1,872 +1,853 @@
       required:
-        - kind
-        - id
-        - inputToken
-        - outputToken
-        - inputAmount
-        - outputAmount
+      - kind
+      - id
+      - inputToken
+      - outputToken
+      - inputAmount
+      - outputAmount
       properties:
-        kind:
-          type: string
-          enum: [ liquidity ]
         id:
-          description: |
-            The ID of executed liquidity provided in the auction input.
-          type: number
-        inputToken:
-          $ref: "#/components/schemas/Token"
-        outputToken:
-          $ref: "#/components/schemas/Token"
+          type: string
+          description: The ID of executed liquidity provided in the auction input.
         inputAmount:
-          $ref: "#/components/schemas/TokenAmount"
+          $ref: '#/components/schemas/TokenAmount'
+        inputToken:
+          $ref: '#/components/schemas/Token'
+        kind:
+          type: string
+          enum:
+          - liquidity
         outputAmount:
-          $ref: "#/components/schemas/TokenAmount"
-
-    Allowance:
-      description: |
-        An ERC20 allowance from the settlement contract to some spender that is
-        required for a custom interaction.
-      type: object
-      required:
-        - token
-        - spender
-        - minAmount
-      properties:
-        token:
-          $ref: "#/components/schemas/Token"
-        spender:
-          $ref: "#/components/schemas/Address"
-        amount:
-          $ref: "#/components/schemas/TokenAmount"
-
-    CustomInteraction:
-      description: |
-        A searcher-specified custom interaction to be included in the final
-        settlement.
+          $ref: '#/components/schemas/TokenAmount'
+        outputToken:
+          $ref: '#/components/schemas/Token'
+    LiquidityParameters:
+      oneOf:
+      - $ref: '#/components/schemas/ConstantProductPool'
+      - $ref: '#/components/schemas/WeightedProductPool'
+      - $ref: '#/components/schemas/StablePool'
+      - $ref: '#/components/schemas/ConcentratedLiquidityPool'
+      - $ref: '#/components/schemas/ForeignLimitOrder'
+    NativePrice:
+      type: string
+      description: |-
+        The price in wei of the native token (Ether on Mainnet for example) to buy
+        10**18 of a token.
+      example: '1234567890'
+    Order:
       type: object
+      description: CoW Protocol order information relevant to execution.
       required:
-        - kind
-        - target
-        - value
-        - calldata
-        - inputs
-        - outputs
+      - uid
+      - sellToken
+      - buyToken
+      - sellAmount
+      - fullSellAmount
+      - buyAmount
+      - fullBuyAmount
+      - validTo
+      - kind
+      - receiver
+      - owner
+      - partiallyFillable
+      - preInteractions
+      - postInteractions
+      - sellTokenSource
+      - buyTokenDestination
```

Comment:
This is new because we forgot to alter it in the https://github.com/cowprotocol/services/pull/2620